packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -512,3 +441,47 @@ export class ApiReferencesTable extends React.Component<IApiReferencesTableProps
     );
   };
 }
+
+/**
+ * Loops through text and places code blocks in code elements
+ */
+function _extractCodeBlocks(text: string): React.ReactNode[] {
+  // Unescape some characters
+  text = text.replace(/\\([@<>{}])/g, '$1');
+
+  let regexResult: RegExpExecArray | null;
+  const codeBlocks: { index: number; text: string }[] = [];
+  while ((regexResult = backticksRegex.exec(text)) !== null) {
+    codeBlocks.push({
+      index: regexResult.index,
+      text: regexResult[0]
+    });
+  }
+  if (!codeBlocks.length) {
+    return [text];
+  }
+
+  const textElements: React.ReactNode[] = [];
+
+  let codeIndex = 0;
+  let textIndex = 0;
+  let key = 0;
+  while (textIndex < text.length && codeIndex < codeBlocks.length) {
+    const codeBlock = codeBlocks[codeIndex];
+    if (textIndex < codeBlock.index) {
+      const str = text.substring(textIndex, codeBlock.index);
+      textElements.push(str);
+      textIndex += str.length;
+    } else {
+      textElements.push(<code key={key}>{codeBlock.text.substring(1, codeBlock.text.length - 1)}</code>);
+      codeIndex++;
+      textIndex += codeBlock.text.length;
+    }
+    key++;
+  }
+  if (textIndex < text.length) {
+    textElements.push(text.substring(textIndex));
+  }
+
+  return textElements;
```

Comment:
Regex seems like the wrong tool here; I think it would be more efficient just to walk the text. 
I believe this does pretty much the right thing:

```
const _extractCodeBlocks = (text: string): React.ReactNode[] => {
  const result: React.ReactNode[] = [];
  if (!text || text.length === 0) {
    return result;
  }
  let index = 0;
  let inCodeBlock = false;
  while (index < text.length) {
    let sectionEnd = text.indexOf("`", index);
    if (sectionEnd === -1) sectionEnd = text.length;
    const elementType = inCodeBlock ? "code" : "span";
    result.push(<elementType key={index}>{text.substring(index, sectionEnd).trim()}</elementType>
    inCodeBlock = !inCodeBlock;
    index = sectionEnd + 1;
  }
  return result;
};
```

Reply:
The regex logic was just moved from another part of the file. But I could change to this since I'm guessing it would be more efficient.